.github/workflows/test.yaml

```
@@ -256,4 +260,4 @@ jobs:
                 issue_number: context.issue.number,
                 body: comment
               });
-            }
+            }
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)